packages/calypso-e2e/src/lib/pages/themes-page.ts

```diff
@@ -68,14 +73,16 @@ export class ThemesPage extends BaseContainer {
 	 * To match exactly, supply the full name of the theme as it appears to the user.
 	 * For a fuzzy match, supply partially the name of the them that should be matched.
 	 *
+	 * The first available theme is always returned.
+	 *
 	 * Example:
 	 * 		partial match: `Twenty Twen` -> [Twenty Twenty, Twenty Twenty-One]
 	 * 		exact match: `Twenty Seventeen` -> Twenty Seventeen
 	 *
 	 * @param {string} [name] Theme name to select.
-	 * @returns {Promise<void>} No return value.
+	 * @returns {Promise<ElementHandle>} Reference to the selected theme card on the gallery.
```

Comment:
`select` method is now only for obtaining the ElementHandle to a theme matching the criteria where previously it performed additional actions (clicking into the theme to show details).